ext/crates/fp/src/vector_inner.rs

```diff
@@ -9,6 +9,9 @@ use std::cmp::Ordering;
 use std::convert::TryInto;
 use std::sync::Once;
 
+#[cfg(target_feature = "avx")]
```

Comment:
We would also have to guard this on the target architecture being x86_64.

Reply:
I thought the AVX instruction sets were only available on x86_64, but you're right that we should make sure. It might change in the future.